src/components/download/DownloadPanel.js

```diff
@@ -0,0 +1,57 @@
+import cx from 'classnames'
```

Comment:
I feel like this file should be called "MapInfo" similar

Reply:
Renamed to `DownloadMapInfo`